src/FileSystem/SourceFileCollector.php

```diff
@@ -67,21 +63,9 @@ public function collectFiles(
             ->exclude($excludeDirectories)
             ->in($sourceDirectories)
             ->files()
+            ->name('*.php')
         ;
 
-        if ($filter === '') {
-            $finder->name('*.php');
-        } else {
-            $finder->filterFiles(
```

Comment:
1. Am I right saying this moving won't impact performance / memory usage? Is it possible that any code in the middle (between `SourceFileCollector` and `FilteredEnrichedTraceProvider` can use this files results, without filters applied?

2. Seems like `FilterableFinder::filterfiles()` is no longer used (and `FilterableFinder` in general)

Reply:
1. FilterableFinder uses the same RealPathFilterIterator we used in `FilteredEnrichedTraceProvider` before to filter coverage-sourced files, therefore we have no reason to believe the performance will be affected. 
2. `FilterableFinder` is indeed no longer needed. I'll remove it.

Reply:
Sorry, for performance question, I meant this

> Is it possible that any code in the middle (between `SourceFileCollector` and `FilteredEnrichedTraceProvider` can use these file results, without filters applied?

Just want to be sure not filerered collection of files won't be used anywhere before `FilteredEnrichedTraceProvider`

Reply:
I made `SourceFileCollector` return a generator, so we can be 100% certain it is not going to be used more than once.

Reply:
👍